Vagrantfile

```diff
@@ -32,7 +32,23 @@ Vagrant.configure('2') do |config|
   config.ssh.shell = %{bash -c 'BASH_ENV=/etc/profile exec bash'}
 
   # Required for NFS to work
-  config.vm.network :private_network, ip: vconfig.fetch('vagrant_ip'), hostsupdater: 'skip'
+  if vconfig.fetch('vagrant_ip') == 'dhcp'
+    config.vm.network :private_network, type: 'dhcp', hostsupdater: 'skip'
+
+    cached_addresses = {}
+    config.hostmanager.ip_resolver = proc do |vm, resolving_vm|
```

Comment:
Small thing, but prefix `resolving_vm` with `_` since it's unused

Reply:
Done